app/src/main/java/com/hedvig/app/feature/home/ui/HomeFragment.kt

```diff
@@ -193,6 +186,16 @@ class HomeFragment : Fragment(R.layout.home_fragment) {
         }
     }
 
+    private fun claimStatusCards(successData: HomeQuery.Data) = HomeModel.ClaimStatus(
```

Comment:
Definitely something for a separate PR yes.
But anyway, I was looking into how it'd look like to fix this, and specifically for the HomeQuery.Data type that'd mean we'd need to create a *huge* new type with mappings of every single type into our own type in order to avoid having an Apollo type existing throughout the entire codebase.
Is this something we want to do? Do we have an example of a place we've done that already?